ui/raidboss/data/02-arr/raid/t11.js

```diff
@@ -93,7 +93,7 @@
     },
     {
       id: 'T11 Phase 2',
-      regex: /:Kaliya HP at 60%/,
+      regex: / :Kaliya HP at 60%/,
```

Comment:
I don't think this space should be added, here and on other HP lines.

Reply:
sry for that, reverted the change